docs/docs/hpc-on-k8s/simpy.md

```diff
@@ -0,0 +1,56 @@
+# SimPy Workload
+SimPy is a tool that helps you simulate real-world processes using Python. It's like building a virtual world where you can model how different things interact with each other, such as customers in a store, vehicles on a road, or agents in a network. It works by using Python's generator functions to define the behavior of the different components in your simulation. These components can be anything from individual customers to complex systems like traffic networks.
+
+
+## Steps to Execute Job
+Below is the example manifest file for SimPy workload in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: simpy-
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 1
+  completions: 1
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: shubhamdang/my_simpy_image:latest
```

Comment:
Is this still the image repository or has the openflight one been created now?

Reply:
updated and image is prsent in openflighthpc dockerhub account.